include/circt/Dialect/FIRRTL/FIRRTLStatements.td

```diff
@@ -428,4 +429,26 @@ def LayerBlockOp : FIRRTLOp<
   let hasCanonicalizeMethod = 1;
 }
 
+//===----------------------------------------------------------------------===//
+// Bind Op
+//===----------------------------------------------------------------------===//
+
+def BindOp
+    : FIRRTLOp<"bind", [DeclareOpInterfaceMethods<InnerRefUserOpInterface>]> {
+  let summary = "Indirect instantiation statement";
+  let description = [{
+    Indirectly instantiate a module from the context of another module. BindOp
+    pairs with a `firrtl.instance` which tracks all information except the
+    emission point for the bind.
+
+    See `sv.bind` for the system verilog equivalent.
```

Comment:
Could the PR include a note that this is only used to aid in lowering and for allow for tracking of where a SystemVerilog bind op will be located (in what file). Essentially, include some of the great PR text here about the motivation for this.